lib/supa/builder.rb

```diff
@@ -21,6 +21,12 @@ def initialize(subject, representer:, tree:)
       end
     end
 
+    def attributes(names, options = {})
```

Comment:
I like what @dasnotme suggested so I've used it https://github.com/distribusion/supa/pull/21/commits/8cfbf8b92adf6d60944af5912e60c102b5409a4e. 

@Strech